src/server/Renderers/RenderFactory.cs

```diff
@@ -11,13 +11,11 @@ public class RenderFactory
     private static IRenderer? GetRendererForFileExtension(string fileExtension)
     {
         // Example for selecting different renderers based on MIME type
-        switch (fileExtension)
+        return fileExtension switch
```

Comment:
what is this sorcery? lmao

Reply:
lol - https://learn.microsoft.com/en-US/dotnet/csharp/language-reference/operators/switch-expression

C# getting function-y